acquisitions/src/main/resources/thunderjet/consortia/tenant-utils/tenant.feature

```diff
@@ -0,0 +1,654 @@
+    When method PUT
+    Then status 404
+    And match response == { errors: [{message: 'Object with consortiumId [d9acad2f-2aac-4b48-9097-e6ab85906b25] was not found', type: '-1', code: 'NOT_FOUND_ERROR'}] }
+
+    # attempt to update non-existing tenant
+    Given path 'consortia', consortiumId, 'tenants', '12345'
+    And request { id: '12345', code: 'ABD', name: 'test', isCentral: true }
+    When method PUT
+    Then status 404
+    And match response == { errors: [{message: 'Object with id [12345] was not found', type: '-1', code: 'NOT_FOUND_ERROR'}] }
+
+    # cases for 422
+    # attempt to update the tenant without an id
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { code: 'ABC', name: 'test', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'id' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without a code
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', name: 'test', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'code' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without a name
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', code: 'ABC', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'name' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without isCentral
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', code: 'ABC', name: 'test' }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'isCentral' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant with a name that has length more than 150 characters and a code with more than 3 characters
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And param adminUserId = consortiaAdmin.id
+    And request
+    """
+    {
+      id: '#(centralTenant)',
+      code: 'TTTTTT',
+      name: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl.',
+      isCentral: true
+    }
+    """
+    When method PUT
+    Then status 422
+    And match response.errors[*].message contains ['\'code\' validation failed. size must be between 2 and 5','\'name\' validation failed. size must be between 2 and 150']
+
+  @Positive
+  Scenario: Do PUT the tenant
```

Comment:
PUT tests for tenants also not needed for orders tests, we neel only POSTs to create ECS env

Reply:
Do we need `Soft Delete and verify data`?

Reply:
nvm, answered below

Reply:
done